src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorDocumentOptionsProvider.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.ExternalAccess.Razor;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+{
+    internal sealed class RazorDocumentOptionsProvider : IRazorDocumentOptionsProvider
+    {
+        private readonly RazorLSPClientOptionsMonitor _optionsMonitor;
+
+        public RazorDocumentOptionsProvider(RazorLSPClientOptionsMonitor optionsMonitor)
+        {
+            _optionsMonitor = optionsMonitor;
+        }
+
+        public Task<RazorDocumentOptions> GetDocumentOptionsAsync(Document document, CancellationToken cancellationToken)
+        {
+            // TO-DO: We should switch to a per-document implementation once Razor starts supporting .editorconfig.
```

Comment:
Just to verify, we don't support .editorconfig today, right? If not, is there a work item that currently exists to track its addition?

Reply:
We at least don't do anything to directly support it. If the editor does under the covers it's transparent to us.